R/mlaps.R

```diff
@@ -173,7 +173,7 @@ loop_mlaps <- function(db, cohort = NULL, hours_after_admission = 0, component_w
           INNER JOIN ", admdad_table, " a
             ON l.genc_id = a.genc_id
           WHERE l.test_type_mapped_omop IN (", paste(LAPS_OMOP_CONCEPTS, collapse = ", "), ")",
-          paste0("AND a.", hospital_field, " = '", hospital_id, "'"),
+          paste0("AND l.", hospital_field, " = '", hospital_id, "'"),
```

Comment:
Hey @marchenaromk, the updated code actually doesn't seem to run for me. I tested it on `drm_cleandb_v2_1_3` and tried to run it on a cohort of just 100 genc_ids. I tried several times but the code never finished/showed any progress. The previous version worked fine for me (though it was super slow...). Did the updated code work for you? I'm using `host="geminiaio.smh.smhroot.net"` - not sure if that might make a difference?

Reply:
@marchenaromk: Ok so based on the discussion in research-roundtable, it sounds like it's fine. I'll go ahead and approve the pull request. You can go ahead and merge once you've resolved the merge conflicts.